gulpfile.js

```diff
@@ -2,16 +2,20 @@ let gulp = require('gulp');
 let cleanCSS = require('gulp-clean-css');
 let imagemin = require('gulp-imagemin');
 
-gulp.task('minify-css', () => {
-    return gulp.src('assets/stylesheets/*.css')
+gulp.task('minify-css', () =>
+    gulp.src('assets/stylesheets/*.css')
         .pipe(cleanCSS())
-        .pipe(gulp.dest('stylesheets'));
-});
+        .pipe(gulp.dest('stylesheets'))
+);
```

Comment:
Why are there semicolons? :) Let's use [standard](https://standardjs.com/) for all code.

Reply:
Death to all semicolons 😉